app/preview/template.njk

```diff
@@ -12,6 +13,9 @@
     {{ viewSolutionDescription({
       section: sections['solution-description']
     }) }}
+    {{ viewSolutionCapabilities({
+      section: sections.capabilities
+    }) }}
```

Comment:
Do we need some UI tests for the capability section?

Reply:
I'll take it out